application.go

```diff
@@ -180,6 +181,13 @@ func (r *Application) CPU(cpu float64) *Application {
 	return r
 }
 
+// GPU set the amount of GPU per instance which is assigned to the application
+//		gpu:	the GPU (check MESOS docs) per instance
+func (r *Application) GPU(gpu float64) *Application {
+	r.GPUs = gpu
+	return r
+}
+
```

Comment:
Could you please also add an `EmptyGPUs` method like this:

```go
func (r *Application) EmptyGPUs() *Application {
	g := 0.0
	r.GPUs = &g
	return r
}
```

This will allow users to reset the GPUs parameter to 0.

See similar functions for how to include this in your test.